[2024-08-22 22:31:52,571] 187 werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://10.4.198.189:5000
[2024-08-22 22:31:52,571] 187 werkzeug - INFO - [33mPress CTRL+C to quit[0m
[2024-08-22 22:32:02,552] 187 werkzeug - INFO - 127.0.0.1 - - [22/Aug/2024 22:32:02] "GET /predictdata HTTP/1.1" 200 -
[2024-08-22 22:32:36,891] 1744 app - ERROR - Exception on /predictdata [POST]
Traceback (most recent call last):
  File "C:\Users\HP\OneDrive\Desktop\New folder\Lib\site-packages\flask\app.py", line 2529, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\HP\OneDrive\Desktop\New folder\Lib\site-packages\flask\app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\HP\OneDrive\Desktop\New folder\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\HP\OneDrive\Desktop\New folder\Lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\credit risk\app.py", line 30, in predict_datapoint
    loan_status = int(request.form.get('loan_status')),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
[2024-08-22 22:32:36,954] 187 werkzeug - INFO - 127.0.0.1 - - [22/Aug/2024 22:32:36] "[35m[1mPOST /predictdata HTTP/1.1[0m" 500 -
